src/cljs/app/views.cljs

```diff
@@ -1,25 +1,33 @@
 (ns app.views
-  "Contains the views, exceot the parent view."
+  "Contains the views, except the root view."
   (:require [cljsjs.reactable]
             [re-frame.core :as re-frame]
-            [reagent.core :as reagent]))
+            [reagent.core :as reagent]
+            [secretary.core :as secretary]))
 
 (def Table js/Reactable.Table)
+(def Tr js/Reactable.Tr)
 
-(defn search-results
+;;;;;;;;;;;;;;;;;;
+;; Search Panel ;;
+;;;;;;;;;;;;;;;;;;
```

Comment:
This usually is indication enough to move the functions in the section to a separate namespace.